### PR TITLE
[docs] PR for Document new LDAP changes for platform

### PR DESCRIPTION
**Pull request by lizayugabyte**
Addresses https://yugabyte.atlassian.net/browse/PLAT-5086

### CODE REVIEW

File: docs/content/preview/yugabyte-platform/administer-yugabyte-platform/ldap-authentication.md
Comment (TaranSinghania) [inline docs/content/preview/yugabyte-platform/administer-yugabyte-platform/ldap-authentication.md:107]: Can you just remove the `,` before `DC=yugabyte,DC=com`? We don't need that anymore. Thanks!

**Review by TaranSinghania — APPROVED** [review]
Approving to unblock. Could you address the comment before merging?
Other changes LGTM. Thanks!

File: docs/content/preview/yugabyte-platform/administer-yugabyte-platform/ldap-authentication.md
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/administer-yugabyte-platform/ldap-authentication.md:107]: Fixed

**Pull request merged**
